plans/extend_ca_cert.pp

```diff
@@ -40,6 +40,14 @@
     }
   }
 
+  if $is_pe {
+    $crl_results = run_task('ca_extend::check_crl_cert', $targets, '_catch_errors' => true).first
```

Comment:
Just one minor thing here, we probably want to run the `check_crl_cert` and `crl_truncate` tasks without the `'_catch_errors'` option.  If either of those fail, then we want the overall plan to fail and return the error from the tasks.  There are some other examples here, like stopping services or running the `extend_ca_cert` task.

After that it's good to go, and someone else can approve and merge if needed because I'm on PTO next week.